modules/logging-loki-cli-install.adoc

```diff
@@ -160,7 +160,7 @@ spec:
   size: 1x.small # <3>
   storage:
     schemas:
-    - version: v12
+    - version: v13
```

Comment:
I think I would also adjust the `effectiveDate` to be more recent. Mostly so that if people accidentally overwrite an old configuration they do not retroactively try to access old data with the new schema.

Reply:
I'm replacing `"2022-06-01"` with `    - effectiveDate: "<yyyy>-<mm>-<dd>"`.

> Mostly so that if people accidentally overwrite an old configuration they do not retroactively try to access old data with the new schema.

Please note that this is a docs file for the installation procedure. Do you expect anyone to have an old configuration when installing it?

Reply:
Overall, I think we just need something explaining the concepts of the schema setting. Having a default for fresh installs is not an issue in this case.

We have had a few cases already where customers kept changing the schema version for past dates and wondered why it was not working as they expected.